<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_window</class>
 <widget class="QMainWindow" name="task_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/resourse/user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
    image: url(:/resourse/Button-First-icon.png);
	
}
QPushButton#pushButton:hover{
	
	color: rgba(131,96,53,255);
	
}
QPushButton#pushButton:pressed{
	
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
	
}
</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2{
	
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
    image: url(:/resourse/Button-Last-icon.png);
	
}
QPushButton#pushButton_2:hover{
	
	color: rgba(131,96,53,255);
	
}
QPushButton#pushButton_2:pressed{
	
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
	
}
</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#logout{
	
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
    image: url(:/resourse/logout.png);
	
}
QPushButton#logout:hover{
	
	color: rgba(131,96,53,255);
	
}
QPushButton#logout:pressed{
	
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
	
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>761</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>611</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>задача№3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="exnum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="st2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ответ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>490</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Задача №3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>задача№1</string>
      </property>
     </widget>
     <widget class="QLabel" name="st1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>задача№2</string>
      </property>
     </widget>
     <widget class="QLabel" name="st3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>490</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>300</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>go</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Задача №1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Статиска</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>420</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>update</string>
      </property>
     </widget>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>591</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="stat">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>400</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>460</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Задача №2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Онлайн калькулятор для решение задач по дискретной математики</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>761</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Дано автомат Мура :</string>
        </property>
       </widget>
       <widget class="QLabel" name="variant">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>381</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>691</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resourse/photo_2023-05-11_12-15-47.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>390</y>
          <width>49</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Ответ :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vvodotveta">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>390</y>
          <width>521</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="mess">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>440</y>
          <width>511</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>380</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="generatebutton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>50</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate or Regenerate</string>
        </property>
       </widget>
       <widget class="QLabel" name="check">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>440</y>
          <width>361</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>(Исходная позиция S0)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>49</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Дано автомат Мили  :</string>
        </property>
       </widget>
       <widget class="QLabel" name="millitsk">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>40</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>generate or regenerate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>641</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/newPrefix/resourse/MILI_AUTHOMAT.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ответ :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>390</y>
          <width>491</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>380</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
       <widget class="QLabel" name="mess_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>450</y>
          <width>521</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>(Исходная позиция S1)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>170</y>
          <width>271</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>90</pointsize>
         </font>
        </property>
        <property name="text">
         <string>third</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>63</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Функция №1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>63</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Функция №2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>63</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Функция №3</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="checkk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuOption"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
